src/es_archiver/lib/stats.ts

```diff
@@ -37,6 +37,8 @@ export interface IndexStats {
   };
 }
 
+export type Stats = ReturnType<typeof createStats>;
```

Comment:
There is a very strange pattern using inline class definition to access the parent context in this file... Had to use `ReturnType` to extract a proper TS type.